android/app/src/main/java/org/unicef/ecar/bebbo/ExactAlarmPackage.java

```diff
@@ -0,0 +1,24 @@
+package org.unicef.ecar.bebbo;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ExactAlarmPackage implements ReactPackage {
+
+    @Override
+    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
+        List<NativeModule> modules = new ArrayList<>();
+        modules.add(new ExactAlarmModule(reactContext));
+        return modules;
```

Comment:
You can directly initialize the modules list with a single entry as: 
```
// Return a list containing the ExactAlarmModule
return Collections.singletonList(new ExactAlarmModule(reactContext));
```

Reply:
NO I added the external Native modules so we need to add this